packages/title-case/src/index.ts

```diff
@@ -4,6 +4,7 @@ const IS_MANUAL_CASE = /\p{Ll}(?=[\p{Lu}])/u; // iPhone, iOS, etc.
 const ALPHANUMERIC_PATTERN = /\p{Alphabetic}+/gu;
 const IS_ACRONYM =
   /^(\P{Alphabetic})*(?:\p{Alphabetic}\.){2,}(\P{Alphabetic})*$/u;
+const IS_DIGIT = /\d/u;
```

Comment:
Can you change alphanumeric pattern instead? I think the bug is that it should consider numbers words too.

Reply:
Sure!

Reply:
@blakeembrey I've update the pattern and removed the check for a numeric digit I had added previously. 

Thanks for taking the time to review this PR.